packages/embed/README.md

```diff
@@ -26,3 +26,100 @@ To generate SHA384 for a specific asset from an URL, run the following command i
 ```sh
 curl https://cdn.botframework.com/botframework-webchat/4.3.0/webchat.js | openssl dgst -sha384 -binary | openssl base64 -A
 ```
+
+## Test locally
+
+There are two public surfaces we need to test:
+
+- Embed page, i.e. https://webchat.botframework.com/embed/your-bot-id
+- Azure Portal "Test in Web Chat" blade
+
+### Testing the embed page
+
+In order to test your local build, you can use Fiddler to modify the traffic to serve the Gemini page locally.
+
+Append the following code in FiddlerScript `OnBeforeRequest` function.
+
+```js
+if (oSession.uriContains('https://webchat.botframework.com/embed/your-bot-id/gemini')) {
+    oSession['x-replywithfile'] = 'C:\\Users\\JohnDoe\\Source\\Repos\\BotFramework-WebChat\\packages\\embed\\dist\\index.html';
+    oSession["ui-backcolor"] = "Yellow";
+}
+```
+
+### Testing on Azure Portal
+
+Until we move everyone to the new embed page, your bot might not using the new embed page on Azure Portal.
+
+Append the following code in FiddlerScript `OnPeekAtResponseHeaders` function. It will force your bot to use Gemini regardless of the deployment phase.
+
+When combined with the FiddlerScript above, "Test in Web Chat" in Azure Portal will point to the locally-developed version of the Gemini page.
+
+```js
+if (oSession.uriContains("https://webchat.botframework.com/embed/your-bot-id?features=webchatpreview&t=")) {
+    oSession["ui-backcolor"] = "Yellow";
+    oSession.oResponse.headers.HTTPResponseCode = 302;
+    oSession.oResponse.headers.HTTPResponseStatus = '302 Moved';
+    oSession.oResponse.headers.Add('location', oSession.PathAndQuery.replace('your-bot-id?', 'your-bot-id/gemini?b=your-bot-id&v=4.3&'));
+}
+```
+
+## FAQs
+
+These are common questions around the new Gemini page.
+
+### What is the version of Web Chat I am using on my page?
+
+Press <kbd>F12</kbd> and look at the console. It should say something similar to the following:
+
+```
+Web Chat: Selecting version "default" -> "4" -> "4.3" -> "4.3.0".
+```
+
+The line above means, the version specifier is not specified. And Web Chat use "default" version, which forward to major version 4, then minor version 4.3, then patch version 4.3.0.
+
+### Why I am not seeing Web Chat v4 on my page?
+
+Look at the <kbd>F12</kbd> console log and see how version selection is being done.
+
+If you don't see the line, it is probably because the new Gemini page is not rolled out completely. If your bot is in, the URL should become https://webchat.botframework.com/embed/your-bot-id/gemini. If your bot is not in, it will not have `/gemini` appended.
```

Comment:
```suggestion
If you don't see the above line, this probably means that the new Gemini page has not completely rolled out. If your bot is using Gemini, the URL will be https://webchat.botframework.com/embed/your-bot-id/gemini. If your bot is not in, it will not have `/gemini` appended.
```